conftest.py

```diff
@@ -739,7 +739,7 @@ def get_mnist_dataset(load_mnist_dataset, mnist_shape):
     np.testing.assert_array_almost_equal(y_test_mnist_original, y_test_mnist, decimal=3)
 
 
-# ART test fixture to skip test for specific mlFramework values
+# ART test fixture to skip test for all mlFramework values except one
```

Comment:
Can `framework` be a list of mlFramework tags?

Reply:
It could. I was just changing the docstring to match the current functionality, but I agree that it would be more useful that way.

Reply:
It seems I have issues with the shape of the output when using `kerastf` classifier. See the [logs in CI](https://github.com/Trusted-AI/adversarial-robustness-toolbox/runs/1533769285).